resources/views/components/quantity.blade.php

```diff
@@ -0,0 +1,34 @@
+{{--
+This is a special one; $attributes is used twice and should not be used
+for styling! This makes sure any data like v-model, min, step, etc
+is present on the input and the quantity select component.
+--}}
+<quantity-select v-slot="qtySelect" {{ $attributes }}>
+    <div class="flex border">
+        <x-rapidez::button.outline
+            v-on:click.prevent="qtySelect.decrease"
+            v-bind:disabled="!qtySelect.decreasable"
+            :aria-label="__('Decrease')"
+        >
+            <div class="bg-neutral h-0.5 w-3 hover:bg-white"></div>
+        </x-rapidez::button.outline>
+        <input
+            name="qty"
+            type="number"
+            dusk="qty"
+            class="w-14 px-1 text-center"
+            aria-label="@lang('Quantity')"
+            {{ $attributes }}
```

Comment:
As an alternative we could do something like this to avoid the usage of `$attributes` twice:
```
@attributes([
    ':min' => 'qtySelect.min',
    ':max' => 'qtySelect.max',
    ':step' => 'qtySelect.step',
    'v-model.number' => $attributes->get('v-model.number'),
    'v-on:change' => $attributes->get('v-on:change'),
])
```
But this feels pretty repeating and needs to be updated manually when something else is added.